app/views/candidate_interface/personal_statement/_guidance.html.erb

```diff
@@ -1,11 +1,15 @@
-<p class="govuk-body">You should write between 500 and 1000 words for your personal statement.</p>
-<p class="govuk-body">Here are some examples you could write about (they are just a guide, you do not need to write about all of them):</p>
-
-<ul class="govuk-list govuk-list--bullet">
-  <li>skills you have that are relevant to teaching</li>
-  <li>any experience of working with young people</li>
-  <li>for secondary teacher training: your interest in the subject you want to teach</li>
-  <li>your understanding of why teaching is important</li>
-  <li>your reasons for wanting to train to be a teacher</li>
-  <li>any activities you have done that could be relevant to teaching like first aid courses, sports coaching or volunteering</li>
+<p class="govuk-body">You can write about these 3 areas in your personal statement to show how your knowledge, qualifications and experience make you suited to teaching. Include any information that you think supports your application.</p>
+
+<ul class="govuk-list govuk-list--number">
+  <li><p class="govuk-!-font-weight-bold">Eplain why you want to be a teacher.</p></li>
+  <p class="govuk-body">Reflect on what motivates you about a career in teaching.</p>
+
+  <li><p class="govuk-!-font-weight-bold">Describe why you want to teach this subject or specialism. For primary, describe why you want to teach this age group.</p></li>
+  <p class="govuk-body">Highlight your expertise in a subject gained through your qualifications or work experience.</p>
+
+  <li><p class="govuk-!-font-weight-bold">Describe your relevant experience and personal characteristics and explain how they would make you a great teacher.</p></li>
+  <p class="govuk-body">Discuss any relevant experience of working or volunteering with young people and any transferable skills or strengths gained through life experience that would support you in the classroom.</p>
 </ul>
+
+<p class="govuk-body">The most successful personal statements are between 500 and 1,000 words long, so make sure you include enough detail.</p>
+<p class="govuk-body">Read more about <a href="https://getintoteaching.education.gov.uk/how-to-apply-for-teacher-training/teacher-training-personal-statement" class="govuk-link">what to include in your personal statement.</a></p>
```

Comment:
Can we use `govuk_link_to` and put the URL in the translation files please, somewhere around here maybe:

https://github.com/DFE-Digital/apply-for-teacher-training/blob/e3dbaeff47a9ed8888d34da21aa3c0a3d108816b/config/locales/en.yml#L42

This will make URL audits easier and we can reuse the same link in the future.

Also, the `.` shouldn't be linked. 

Should this link open in a new tab?